airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -946,6 +946,18 @@ protected StandardCheckConnectionOutput runCheck(final JsonNode config) throws W
             .run(new StandardCheckConnectionInput().withConnectionConfiguration(config), jobRoot);
   }
 
+  protected StandardCheckConnectionOutput.Status runCheckWithCatchedException(final JsonNode config) {
```

Comment:
why do we need to use this instead of the standard `runCheck`?

Reply:
the plain runCheck method is throwing a module's specific exception.  So we would need to add the whole module to dependency.  This would make the artifact much bigger and potentially could cause a circle dependency. If I may add it as a dependency if that's a case :)